src/ShopBundle/objects/db/TShopOrder.class.php

```diff
@@ -540,6 +543,32 @@ protected function SaveOrderNotificationDataToOrder($bOrderSend, $oMail, $sSendT
         $this->AllowEditByAll(false);
     }
 
+    protected function SetSendOrderNotificationState(bool $orderSend): void
```

Comment:
No hungarian notation should be used for new code.

Reply:
In fact, there is no Hungarian notation, or not?